src/js/edu-benefits/components/RotcScholarship.jsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+
+import ErrorableTextInput from '../../common/components/form-elements/ErrorableTextInput';
+
+import { isValidYear, validateIfDirty, isNotBlank } from '../utils/validations';
+
+export default class RotcScholarship extends React.Component {
+  render() {
+    const { view, onValueChange } = this.props;
+    const scholarship = this.props.data;
+    const formFields = (
+      <div>
+        <ErrorableTextInput
```

Comment:
Might be worth asking if any of these fields should be required. I think we normally have at least one in these sorts of lists.